athena-federation-sdk/pom.xml

```diff
@@ -287,7 +287,7 @@
         <dependency>
             <groupId>org.apache.commons</groupId>
             <artifactId>commons-lang3</artifactId>
-            <version>3.14.0</version>
+            <version>3.12.0</version>
```

Comment:
Same here; we need to make sure that we aren't using another higher than the dependency used by the engine.